pkg/karmadactl/cmdinit/cert/cert_test.go

```diff
@@ -107,6 +108,32 @@ func TestGenCerts(t *testing.T) {
 		t.Fatal(err)
 	}
 	if err := GenCerts(TestCertsTmp, TestCaCertPath, TestCaKeyPath, etcdServerCertConfig, etcdClientCertCfg, karmadaCertCfg, apiserverCertCfg, frontProxyClientCertCfg); err != nil {
-		fmt.Println(err)
+		t.Fatal(err)
+	}
+
+	certFiles := []string{
+		"apiserver.crt", "apiserver.key",
+		"ca.crt", "ca.key",
+		"etcd-ca.crt", "etcd-ca.key",
+		"etcd-client.crt", "etcd-client.key",
+		"etcd-server.crt", "etcd-server.key",
+		"front-proxy-ca.crt", "front-proxy-ca.key",
+		"front-proxy-client.crt", "front-proxy-client.key",
+		"karmada.crt", "karmada.key",
+	}
+	if err := checkCertFiles(TestCertsTmp, certFiles); err != nil {
```

Comment:
if TestCaCertPath and TestCaKeyPath is not nil and i think this is not TestCertsTmp folder anymore.

Reply:
I don't quite understand the amendment here, could you give me clearer instructions

Reply:
same with https://github.com/karmada-io/karmada/pull/5175#discussion_r1680374431

we need to check two case:

1. TestCaCertPath and TestCaKeyPath  is nil
2. TestCaCertPath and TestCaKeyPath  is not nil

Reply:
I see, I will change it in the latest commit